packages/mui-material/src/Rating/Rating.js

```diff
@@ -498,7 +498,14 @@ const Rating = React.forwardRef(function Rating(inProps, ref) {
       ref={handleRef}
       onMouseMove={handleMouseMove}
       onMouseLeave={handleMouseLeave}
-      className={clsx(classes.root, className)}
+      className={clsx(
```

Comment:
We will need to apply similar patches in `InputBase` as there was quite some time since we added the read only class name.

Reply:
Done.